### PR TITLE
Embeds array for send message async

### PR DESCRIPTION
**Pull request by killerfrienddk**
Since discord has added the support for it in the api I thought why not add it :)

### CODE REVIEW

**Comment by quinchs** [comment]
See #129

**Comment by killerfrienddk** [comment]
@quinchs mine would not break preexisting calls to SendMessageAsync that uses a single embed.
It would allow people to either not change their calls if the list is not needed or change it on their own accord.

**Comment by quinchs** [comment]
@d4n3436 thoughs?

**Comment by d4n3436** [comment]
It would be better to move the  single embed parameter to the end to keep consistency with other methods that also added embed array params (like #72)

Honestly I would just replace the single embed param with the embed array. It will be a breaking change but a better one, since the single embed param will have to be removed eventually.

**Comment by killerfrienddk** [comment]
@d4n3436 This would allow for people not breaking their code if they wish to use the single one then they can it will simply add the single one to the array if the array is null.

**Comment by d4n3436** [comment]
> @d4n3436 This would allow for people not breaking their code if they wish to use the single one then they can it will simply add the single one to the array if the array is null.

Adding an extra parameter anywhere but at the end will be a breaking change anyway.
A better solution may be:

- Keep the embed param in the same place it is right now.
- Add the embed array param at the end.

This solution won't break anyone's code and will allow people to use the embed array param if they want to.

**Comment by killerfrienddk** [comment]
True ill move them to the end.

**Comment by d4n3436** [comment]
Wait that was my idea 😐
nvm

**Comment by killerfrienddk** [comment]
It is done @d4n3436 sorry I had no idea you had a pr, I would just have added the latter stuff to yours if I had known.

File: src/Discord.Net.Commands/ModuleBase.cs [hunk 1]
```diff
@@ -32,16 +32,16 @@ public abstract class ModuleBase<T> : IModuleBase
         /// </param>
         /// <param name="isTTS">Specifies if Discord should read this <paramref name="message"/> aloud using text-to-speech.</param>
         /// <param name="embed">An embed to be displayed alongside the <paramref name="message"/>.</param>
-        /// <param name="allowedMentions">
+        /// <param name="embeds">A array of <see cref="Embed"/>s to send with this response. Max 10</param>        /// <param name="allowedMentions">
```
Comment (MrCakeSlayer) [inline src/Discord.Net.Commands/ModuleBase.cs]: Could you fix the comment lines :)

**Review by MrCakeSlayer — CHANGES_REQUESTED** [review]

**Review by MrCakeSlayer — APPROVED** [review]

**Review by quinchs — APPROVED** [review]

**Pull request merged**